snakemake.el

```diff
@@ -350,6 +350,17 @@ targets."
                                        (snakemake-all-rules))
              " "))
 
+(defun snakemake-graph-workflow ()
+  (interactive)
+  """Creates a graph for the current snakefile."""
```

Comment:
This docstring should be moved up a line and should only consist of
a single set of double quotes.